### PR TITLE
Adds two alternate shuttles

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
rscadd: Nanotrasen is short on cash, and are now borrowing escape shuttles from other stations, and Space Apartments.
/:cl:
wnqbuld
qnr1hwn

### CODE REVIEW

**Comment by ChangelingRain** [comment]
First one should have the docking port pointing south, second should point east.

**Comment by Iamgoofball** [comment]
they already are, i checked the dirs

**Comment by Bawhoppen** [comment]
@Iamgoofball its all changing soon(tm) anyways

**Comment by coiax** [comment]
I'll have a look. Please implement framed pictures, and then implement plastic spoons, and then implement framed pictures of plastic spoons.

**Comment by Anonmare** [comment]
YOU'RE TEARING ME APART GOOF

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -112,6 +112,24 @@
...
+		Oh, pretty good. We got a new client and the bank will make a lot of money. \
+		What client? \
+		I cannot tell you; it's confidential. \
+		Aw, come on. Why not? \
+		No, I can't. Anyway, how is your sex life?"
+
+/datum/map_template/shuttle/emergency/goon
+	suffix = "goon"
+	name = "NES Port"
+	description = "The Nanotrasen Emergency Shuttle Port(NES Port for short) \
```
Comment (coiax) [inline code/datums/shuttles.dm]: Put a space between Port and (

File: code/datums/shuttles.dm [hunk 2]
```diff
@@ -112,6 +112,24 @@
...
+	is a shuttle used at other less known nanotrasen facilities \
```
Comment (coiax) [inline code/datums/shuttles.dm]: Nanotrasen

File: code/modules/shuttle/shuttle.dm [hunk 3]
```diff
@@ -1,5 +1,5 @@
 //use this define to highlight docking port bounding boxes (ONLY FOR DEBUG USE)
-//#define DOCKING_PORT_HIGHLIGHT
+#define DOCKING_PORT_HIGHLIGHT
```
Comment (coiax) [inline code/modules/shuttle/shuttle.dm]: ONLY FOR DEBUG USE

Comment (Iamgoofball) [inline code/modules/shuttle/shuttle.dm]: left it onn for fixing the shuttles

**Comment by coiax** [comment]
Okay, so for some reason the map conversion has fucked up and duplicated some rows, so while @xxalpha  is looking at that, I will tell you what to do.

Oh Hai Mark: mobile docking port, name="Oh Hi Mark", dheight=0, dwidth=1
NES Port: mobile docking port, name="NES Port", dheight=0, dwidth=9

Look at how the d values vary in the standard ferry, standard Box emergency shuttle, and the Efficiency emergency shuttle.

Also, the NES Port has autosmoothed walls which fuck up when shuttles are rotated, so smooth them by hand.

**Comment by Iamgoofball** [comment]
it works now

**Comment by Desucake** [comment]
I am not sure if layout needs to be something to change, but I do think they need to add more useful features to the shuttle, like maybe toxin kits and a medical bot, like the meta station shuttle has.

**Comment by Jalleo** [comment]
@Desucake its that small on purpose. Everyone gets stuck trying to get on its a deathtrap shuttle.

**Comment by coiax** [comment]
> The office shuttle is just so odd, its literally just a wooden box with some sort of bar set up, it had one door, and the place is just very small

[Leave your stupid comments in your pocket!](http://assets1.theroomsoundboard.com/LeaveYourCommentsPocket.mp3)

(Spoiler: [The Room](https://en.wikipedia.org/wiki/The_Room_(film)))

**Comment by coiax** [comment]
Goof, replace the shuttle airlock on Oh Hi Mark with a wooden airlock please.

**Comment by Iamgoofball** [comment]
got it, wasn't certain if it needed to be a shuttle airlock or not, I wanted a wooden door

**Comment by ChangelingRain** [comment]
abusing [s] to hide prs from the players is easily grounds for pr closure

**Comment by coiax** [comment]
Wait [s] actually does something?

**Comment by Jordie0608** [comment]
It stops the PR from being announced on the servers when opened/closed/merged.

**Comment by Iamgoofball** [comment]
i was just pulling a prank and didn't want to spam the servers but they overreacted

**Pull request closed without merging**